packages/bruno-app/src/components/RequestPane/Script/index.js

```diff
@@ -41,26 +41,28 @@ const Script = ({ item, collection }) => {
   return (
     <StyledWrapper className="w-full flex flex-col">
       <div className="flex-1 mt-2">
-        <div className="mb-1 title text-xs">Pre Request</div>
-        <CodeEditor
+        <h3 className="title text-xs mb-2">Pre Request</h3>
```

Comment:
Changed to h3, but the same `Pre request` title is also at least on request Vars tab. So would leave it as it was or change it everywhere to look the same.